lib/api/contact.js

```diff
@@ -0,0 +1,32 @@
+const Request = require('./request')
+
+class Contact extends Request {
+
+  constructor(httpClient){
+    super()
+    this.httpClient = httpClient
+  }
+
+  get(channel_id, contact_id, accessToken = '') {
```

Comment:
pro js kód se v této libce používá camel case. `channel_id` → `channelId` a `contact_id` → `contactId`